massdriver-application-azure-function-app/_variables.tf

```diff
@@ -6,14 +6,24 @@ variable "docker" {
   })
 }
 
-
 variable "application" {
   type = object({
     sku_name             = string
     minimum_worker_count = number
     maximum_worker_count = number
     cidr                 = string
     zone_balancing       = bool
+    health_check_path    = string
+    app_logs = object({
```

Comment:
If we're going to nest logging config inside of `application` we can drop `app`. So it would be `application.logs...` instead of `application.app_logs`